draft-ietf-dnsop-svcb-https.md

```diff
@@ -763,6 +759,33 @@ in the initial response.  As a performance optimization, if some of the SVCB
 records in the response can be used without requiring additional DNS queries,
 the client MAY prefer those records, regardless of their priorities.
 
+# SVCB-compatible 
+
+An RR type is called "SVCB-compatible" if it permits an implementation that is
+identical to SVCB in its:
+
+* RDATA presentation format
+* RDATA wire format
+* IANA registry used for SvcParamKeys
+* Authoritative server Additional Section processing
+* Recursive resolution process
+* Relevant Class (i.e. Internet ("IN") {{!RFC1035}})
+
+This allows authoritative and recursive DNS servers to apply identical
+processing to all SVCB-compatible RR types.
+
+All other behaviors described as applying to the SVCB RR also apply
+to all SVCB-compatible RR types unless explicitly stated otherwise.
+When following an AliasMode record of RR type `T` ({{alias-mode}}), the
+followup query to the TargetName MUST also be for type `T`.
```

Comment:
Responding to https://github.com/MikeBishop/dns-alt-svc/pull/335#r723713663

Is this clearer?
```suggestion
When following an AliasMode record ({{alias-mode}}) of RR type $T , the
followup query to the TargetName MUST also be for type $T.
```